src/opnsense/mvc/app/views/OPNsense/Diagnostics/log.volt

```diff
@@ -151,12 +151,14 @@
           let select = $("#severity_filter");
           let header_val = filter_exact ? m_header : s_header;
 
+          // switch select mode and reinit selectpicker
           select.prop("multiple", filter_exact);
-          // switch select mode and destroy selectpicker
-          if (!select.hasClass('severity_filter_init')) {
-              select.addClass('severity_filter_init');
-          } else {
-              select.selectpicker('destroy');
+          select.selectpicker('destroy');
+          // keep select open after reinit if called from mode switch
+          if (event && (event.currentTarget.id = 'execat_severity')) {
```

Comment:
`exact_severity` ?

Reply:
sorry, sure )
funny but that was copy/paste from chrome console (i thought)